t/cs.t

```diff
@@ -2056,3 +2065,129 @@ GET /c
 failed to receive: failed to receive the first 2 bytes: closed
 --- error_log
 failed to send close: bad status code
+
+
+
+=== TEST 29: mutual TLS with client certs
+--- no_check_leak
+--- http_config eval: $::HttpConfig
+--- config
+    listen 12345 ssl;
```

Comment:
Please do not use port 12345. This port is blocked by iptables in other lua-nginx-module.

we can use $TEST_NGINX_RAND_PORT_1, please refer to 
https://github.com/openresty/lua-nginx-module/blob/master/t/139-ssl-cert-by.t#L459